core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/LoginViewModel.kt

```diff
@@ -0,0 +1,17 @@
+package com.ustadmobile.core.viewmodel
+
+data class LoginUiState(
+    val username: String = "",
+    val password: String = "",
+    val usernameEnabled: Boolean = true,
+    val passwordEnabled: Boolean = true,
+    val isEmptyPassword: Boolean = false,
+    val usernameError: String? = null,
+    val passwordError: String? = null,
+    val versionInfo: String = "v42",
+    val isEmptyUsername: Boolean = false,
+    val inProgress: Boolean = false,
```

Comment:
We don't need the inProgress property on Jetpack compose. We can just keep the fieldsEnabled property.